azure/pas-single-component-deploy.json

```diff
@@ -89,19 +89,23 @@
                 "description": "Enter the Subnet name."
             }
         },
-        "Component AvailablitySet Name": {
-            "type": "String",
-            "metadata": {
-                "description": "Enter the Availability Set for the Component VM , If doesn’t exist it will be created"
+        "Availability Zone":{
+            "type":"string",
+            "allowedValues": [
+                "1",
+                "2",
```

Comment:
4 is not allowed?

Reply:
it is allowed but there are regions where 4 isnt available. we decided 3 is more than enough